lib/routes/nautiljon/manga-releases.ts

```diff
@@ -0,0 +1,70 @@
+import { load } from 'cheerio';
+import { Route } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import puppeteer from '@/utils/puppeteer';
+import logger from '@/utils/logger';
+
+const host = 'https://www.nautiljon.com';
+export const route: Route = {
+    path: '/releases/manga',
+    categories: ['reading'],
+    example: '/releases/manga',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['nautiljon.com/'],
+        },
+    ],
+    name: 'Nautiljon France manga releases',
```

Comment:
```suggestion
    name: 'France manga releases',
```
No need to include namespace name 
https://github.com/DIYgod/RSSHub/blob/bacaf639aa2f06a4ec68bdb6b26669e567eb90a9/lib/routes/nautiljon/namespace.ts#L4 in route name